torch_xla/csrc/init_python_bindings.cpp

```diff
@@ -160,6 +160,22 @@ class PythonScope : public Scope {
   };
 };
 
+static void ConsumeAndMaybeThrow(absl::Status status) {
+  if (status.ok()) {
+    return;
+  }
+
+  throw std::runtime_error(std::string(status.message()));
+}
+
+template <class T>
+static T ConsumeAndMaybeThrow(absl::StatusOr<T> status) {
```

Comment:
If you call `status.value()`, it will throw if the value is not present, so I'm not sure what value this adds.

Reply:
I think that it would be better to have one source of truth for raising exceptions from these status constructs. In other words, always call `ConsumeAndMaybeThrow` whenever we want to do that. What do you think?

Reply:
My idea was to have one source of truth for both `Status` and `StatusOr<T>` types. Since `Status.value()` doesn't exist, I thought that it would be better to have a single `ConsumeAndMaybeThrow()` function for both of them. Then, it would be easier to spot whenever we are transitioning from `Status*` to throwing exceptions.

Reply:
SG